templates/zerver/integrations/email.md

```diff
@@ -1,3 +1,5 @@
+# Zulip Email integration
```

Comment:
Do we want to wrap these in tabs?
To look something like this?
![image](https://github.com/user-attachments/assets/6bfc9446-7338-4e28-a9d2-3036574f6c75)

Reply:
Hmmm... That's more of a description vs instruction. I'd leave it out of the block as it is, and have the instruction block be something like ...

```
1. See our help center documentation for [messaging a channel by email](/help/message-a-channel-by-email).
```

Reply:
I've removed the use cases part from the Email integration, since it's a duplicate of what's mentioned in the help center article.